src/estimation/include/iDynTree/Estimation/BerdyHelper.h

```diff
@@ -677,17 +776,20 @@ class BerdyHelper
      *       are invariant with respect to an offset in linear velocity. This convenient to avoid
      *       any dependency on any prior floating base estimation.
      *
+     * @param[in] baseTransform the transformation from floating base to world
```

Comment:
This line does not seems to correspond to an argument that is actually passed to the function.